docs/docs/data-structures/tree/index.mdx

```diff
@@ -20,467 +18,45 @@ A `SharedTree` has the following characteristics:
-```
-
-Inserts the provided value(s) at the specified `index`. If the `index` is greater than the length of the array, the items are inserted at the end of the array.
-
-```typescript
-insertAtStart(value: Iterable<T>)
-```
-
-Inserts the provided value(s) at the start of the array. This is sugar for `insertAt(0, …)`.
-
-```typescript
-insertAtEnd(value: Iterable<T>)
-```
-
-Inserts the provided value(s) at the end of the array. This is syntactic sugar for `insertAt(Infinity, …)`.
-
-###### Remove methods
-
-Array nodes have two methods that remove items from the node. Note the following about these methods:
-
--   Removed items are saved internally for a time in case they need to be restored as a result of an undo operation.
--   A removed item may be restored as a result of a simultaneous move operation from another client. For example, if one client removes items 3-5, and another client simultaneously moves items 4 and 5, then, if the move operation is ordered last, only item 3 is removed (items 4 and 5 are restored and moved to their destination by the move operation). If the remove operation is ordered last, then all three items will be removed, no matter where they reside.
--   Removal of items never overrides inserting (or moving in) items. For example, if one client removes items 10-15, and another client simultaneously inserts an item at index 12, the original items 10-15 are removed, but new item is inserted between item 9 and the item that used to be at index 16. This is true regardless of the order of the remove and insert operations.
-
-For the meaning of "simultaneously", see [Types of distributed data structures](../overview.mdx).
-
-```typescript
-removeAt(index: number)
-```
-
-Removes the item at the given `index`.
-
-```typescript
-removeRange(start?: number, end?: number)
-```
-
-Removes the items indicated by the `start` index (inclusive) and `end` index (exclusive). If the end index is omitted, every item from the start index to the end of the array is removed. If the start index is omitted, it defaults to 0. So, calling `removeRange()` removes all the items in the array.
-
-###### Move methods
+To get started working with `SharedTree` in your application, read this [quick start guide](../../start/tree-start.mdx).
+The major programming tasks for using `SharedTree`s that are described in this doc are:
 
-Array nodes have three methods that move items within an array or from one array node to another. When moving from one array node to another, these methods must be called from the destination array node. Note that in all of the following, the `T` can be any type that is derived from an object that is returned by a call of `SchemaFactory.array()`, such as the `Notes` and `Items` classes in the sticky notes example.
+-   Defining a schema for the `SharedTree`. As you build out from prototype to full production application, you can add to this schema and create additional schemas for additional `ShareTree`s.
```

Comment:
"additional `ShareTree`s" sounds like separate ST instance. While that is indeed possible, that's not something we want to encourage so it probably best not to mention it in such a prominent place. We don't want to give the impression that having multiple STs is a common/standard thing.